Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/RATestProcessor.kt

```diff
@@ -276,3 +261,19 @@ private fun CoronaTestResult.toValidatedResult(): CoronaTestResult {
         RAT_INVALID
     }
 }
+
+// After 60 days, the previously EXPIRED test is deleted from the server, and it may return pending again.
+fun check60DaysRAT(test: BaseCoronaTest, newResult: CoronaTestResult, now: Instant): CoronaTestResult {
+    val testAge = Duration(test.registeredAt, now)
+    Timber.tag(RATestProcessor.TAG)
+        .d("Calculated test age: %d days, newResult=%s", testAge.standardDays, newResult)
+
+    return if ((newResult == PCR_OR_RAT_PENDING || newResult == RAT_PENDING) &&
+        testAge > VerificationServer.TestAvailabilityDuration
+    ) {
+        Timber.tag(RATestProcessor.TAG).d("$testAge is exceeding the test availability.")
```

Comment:
Dont use string templates for logging with timber. Make use of the lazy loading.
And VerificationServer.TestAvailabilityDuration should be logged to

Reply:
Ok